php_sql/task1/electives.sql

```diff
@@ -0,0 +1,34 @@
+
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+
+CREATE TABLE `electives` (
```

Comment:
I like that you create the table in the sql script, but it will be even better if you create the database, too.